File: packages/insomnia-httpsnippet/test/fixtures/output/c/libcurl/headers.c
CURL *hnd = curl_easy_init();

curl_easy_setopt(hnd, CURLOPT_CUSTOMREQUEST, "GET");
curl_easy_setopt(hnd, CURLOPT_URL, "http://mockbin.com/har");

struct curl_slist *headers = NULL;
headers = curl_slist_append(headers, "x-foo: Bar");
headers = curl_slist_append(headers, "accept: application/json");
curl_easy_setopt(hnd, CURLOPT_HTTPHEADER, headers);

CURLcode ret = curl_easy_perform(hnd);

File: packages/insomnia-httpsnippet/test/fixtures/output/javascript/fetch/headers.js
const fetchOptions = {
  mode: 'cors',
  method: 'GET',
  headers: {
    accept: 'application/json',
    'x-foo': 'Bar',
  },
};

fetch('http://mockbin.com/har', fetchOptions)
  .then(response => response.json())
  .then(data => console.log(data));
